__tests__/app.test.js

```diff
@@ -81,6 +81,36 @@ describe("GET /api/articles", () => {
   });
 });
 
+describe("GET /api/articles (queries)", () => {
+  it("returns 200 and articles with a topic of 'mitch'", () => {
+    return request(app)
+      .get("/api/articles?topic=mitch")
+      .expect(200)
+      .then(({ body }) => {
+        body.articles.map((article) => {
+          expect(article).toHaveProperty("topic");
+          expect(article.topic).toBe("mitch");
+        });
+      });
```

Comment:
a forEach would be better suited here, just make sure to check the length before. A map produces a new array, which shouldn't really be happening in tests.